src/Core/TileMesh.js

```diff
@@ -34,6 +34,7 @@ class TileMesh extends THREE.Mesh {
         this.rotationAutoUpdate = false;
 
         this.layerUpdateState = {};
+        this.isTileMesh = true;
```

Comment:
Do the appropriate change in `convertToTile` and `Picking` please.

Reply:
yep